### PR TITLE
feat:added Hero section

### PR DESCRIPTION
**Pull request by aryab2003**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
 [![Starware](https://img.shields.io/badge/Starware-⭐-black?labelColor=f9b00d)](https://github.com/zepfietje/starware)
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
```
Comment (Shurtu-gal) [inline README.md:6]: Why is this duplicated?

File: package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
 		"react": "^18.2.0",
 		"react-dom": "^18.2.0",
 		"react-router-dom": "^6.21.3",
+		"styled-components": "^6.1.8",
```
Comment (Shurtu-gal) [inline package.json:23]: Can be removed if not used.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Srish-ty) [reply README.md:5]: fixed. [changes](https://github.com/dscnitrourkela/project-udon/pull/17/commits/25b1a1a1ed993d4a08e4fba0e90c16de2898e65d)

File: package.json (hunk 2 above)
Comment (Srish-ty) [reply package.json]: done. [commit](https://github.com/dscnitrourkela/project-udon/pull/17/commits/25b1a1a1ed993d4a08e4fba0e90c16de2898e65d)

File: src/components/Hero.jsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+						<Paragraph style={{ fontSize: '75px', fontWeight: '500', letterSpacing: '0.75px', lineHeight: 'normal' }}>
+							Lorem Ipsum Dolar sit Amet Luyto
+						</Paragraph>
+					</div>
+					<div className='h-1/4'>
+						<button
+							className='px-4 py-2 rounded-md border border-solid border-black bg-orange-500 shadow-button mt-20 md:mt-35'
+							style={{
+								boxShadow: '2px 2px 0px 0px #000, 3px 4px 9.2px 0px rgba(222, 222, 222, 0.48) inset',
+								fontFamily: 'Syne, sans-serif',
```
Comment (Shurtu-gal) [inline src/components/Hero.jsx:22]: Please use typography here instead of defining styles.

File: src/components/Hero.jsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+				<div className='w-[100%] md:w-1/2 '>
+					<div className='h-1/3 flex flex-col justify-end ml-2' style={{ color: '#FF4409' }}>
+						<PersonalizedText>Welcome family...</PersonalizedText>
+					</div>
+					<div className='h-1/3'>
+						<Paragraph style={{ fontSize: '75px', fontWeight: '500', letterSpacing: '0.75px', lineHeight: 'normal' }}>
+							Lorem Ipsum Dolar sit Amet Luyto
+						</Paragraph>
+					</div>
+					<div className='h-1/4'>
```
Comment (Shurtu-gal) [inline src/components/Hero.jsx:17]: Need of div here?

File: src/components/Hero.jsx [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+								// width: '275px',
+								// height: '320px',
+								// marginLeft: '12px',
+								// marginTop: '10px',
+							}}></div>
+					</div>
+					<div
+						className='absolute md:h-[414px] md:w-[300px] w-[210px] h-[278px]'
+						style={{
+							borderRadius: '6px',
```
Comment (Shurtu-gal) [inline src/components/Hero.jsx:64]: Can use tailwind for these things.

File: src/components/Hero.jsx [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+							// height: '325.558px',
+							transform: 'rotate(15.801deg)',
+							flexShrink: 0,
+							left: '45%',
+							zIndex: 1,
+						}}>
+						<div
+							className='md:w-[275px] md:h-[257.4px] w-[175px] h-[206px] md:mr-[10px] mr-[15px] md:ml-[12px] ml-[6px]'
+							style={{
+								backgroundImage: "url('https://res.cloudinary.com/dkpdi1wek/image/upload/v1709662873/right_guw2gk.jpg')",
```
Comment (Shurtu-gal) [inline src/components/Hero.jsx:145]: Put this in data.

File: src/components/Hero.jsx [hunk 7]
```diff
@@ -0,0 +1,161 @@
+import { PersonalizedText } from './shared/typography';
+import { Paragraph } from './shared/typography';
+
+const Hero = () => {
+	return (
+		<>
+			<div className='flex flex-col md:flex-row'>
+				<div className='w-[100%] md:w-1/2 '>
+					<div className='h-1/3 flex flex-col justify-end ml-2' style={{ color: '#FF4409' }}>
```
Comment (Shurtu-gal) [inline src/components/Hero.jsx:9]: What's the use setting height here?

**Review by Shurtu-gal — DISMISSED** [review]
Fix this:-

image

File: src/components/Hero.jsx (hunk 4 above)
Comment (aryab2003) [reply src/components/Hero.jsx:17]: For the button container

File: src/components/Hero.jsx (hunk 7 above)
Comment (aryab2003) [reply src/components/Hero.jsx:9]: so as to divide into 3 sub divs under the left main div

File: src/components/Hero.jsx [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+	return (
+		<>
+			<div className='flex flex-col md:flex-row'>
+				<div className='w-[100%] md:w-1/2 '>
+					<div className='h-1/3 flex flex-col justify-end ml-2' style={{ color: '#FF4409' }}>
+						<PersonalizedText>Welcome family...</PersonalizedText>
+					</div>
+					<div className='h-1/3'>
+						<Paragraph style={{ fontSize: '75px', fontWeight: '500', letterSpacing: '0.75px', lineHeight: 'normal' }}>
+							Lorem Ipsum Dolar sit Amet Luyto
```
Comment (Shurtu-gal) [inline src/components/Hero.jsx:14]: Half of the text is using this https://github.com/dscnitrourkela/project-udon/blob/a853ba0bdf12959d6fa6d0be3f2fb76dacdc52cf/src/components/shared/typography/index.jsx#L7

File: src/components/Hero.jsx (hunk 3 above)
Comment (Srish-ty) [reply src/components/Hero.jsx]: changed https://github.com/dscnitrourkela/project-udon/pull/17/commits/537e17d76660156681afe3dabb9cec6f44c6c672

File: src/components/Hero.jsx (hunk 5 above)
Comment (Srish-ty) [reply src/components/Hero.jsx]: fixed https://github.com/dscnitrourkela/project-udon/pull/17/commits/458135db8b03d72a81e6483df3a32f7b7154882b

File: src/components/Hero.jsx (hunk 7 above)
Comment (Srish-ty) [reply src/components/Hero.jsx:10]: @aryab2003 need to use flex for that

Comment (Srish-ty) [reply src/components/Hero.jsx]: fixed https://github.com/dscnitrourkela/project-udon/pull/17/commits/3744a23efb73bcf7bc73ba221099c39ba27c9082

File: src/components/Hero.jsx (hunk 6 above)
Comment (Srish-ty) [reply src/components/Hero.jsx]: done. https://github.com/dscnitrourkela/project-udon/pull/17/commits/3aa5dffee4f86b7f2fa359f6d122085d99adc800

File: src/components/Hero.jsx [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+								strokeWidth: '1.2px',
+								stroke: '#000',
+							}}>
+							<path
+								d='M1.4381 56.106C1.43808 56.106 1.43806 56.106 1.43804 56.106C0.401629 55.984 0.399996 54.4803 1.43614 54.3561C1.43616 54.3561 1.43618 54.3561 1.4362 54.3561L16.0482 52.6062C35.1916 50.3137 50.2703 35.2022 52.5212 16.054L54.2394 1.43818C54.3613 0.401548 55.8651 0.399915 55.9893 1.43633L57.7391 16.0483C60.0317 35.1916 75.1432 50.2703 94.2914 52.5212L108.907 54.2393C109.944 54.3613 109.945 55.865 108.909 55.9892L94.297 57.7391C75.1538 60.0316 60.0751 75.1431 57.8241 94.2915L56.106 108.907C56.106 108.907 56.106 108.907 56.106 108.907C55.984 109.944 54.4804 109.945 54.3561 108.909C54.3561 108.909 54.3561 108.909 54.3561 108.909L52.6062 94.2972C50.3137 75.1537 35.2022 60.075 16.0539 57.8241L1.4381 56.106Z'
+								fill='#FFE500'
+								stroke='black'
+								strokeWidth='1.2'
+							/>
+						</svg>
```
Comment (Shurtu-gal) [inline src/components/Hero.jsx:80]: Use a image link like this https://res.cloudinary.com/dkivz06xv/image/upload/v1710089645/rvqsfd4sqaclbthidnxl.png

image

Comment (aryab2003) [reply src/components/Hero.jsx:80]: okay

**Comment by Shurtu-gal** [comment]
@aryab2003 make the images responsive also currently they are hogging all the space.

**Review by ashutosh-rath02 — APPROVED** [review]

**Comment by Shurtu-gal** [comment]
@all-contributors add @aryab2003 for code

**Pull request merged**